src/Components/Admins/Modal/modal.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import modalstyles from './modal.module.css';
+
+const Modal = ({ content, contentMessage, title, setModalDisplay }) => {
+  return (
+    <>
+      <div
+        id="id-screen"
+        onClick={() => setModalDisplay(false)}
+        className={modalstyles.screen}
+      ></div>
+      <div id="id-modal" className={modalstyles.modal}>
+        <header className={modalstyles.header}>
+          <h3 className={modalstyles.header__title}>{title}</h3>
+          <button className={modalstyles.header__button} onClick={() => setModalDisplay(false)}>
+            <i className="fa-solid fa-square-xmark fa-2xl"></i>
+          </button>
+        </header>
+        <div className={modalstyles.content}>
+          {content ?? null}
+          {contentMessage ? <p className={modalstyles.content__message}>{contentMessage}</p> : null}
+          <button
+            className={`${modalstyles.options__button} ${modalstyles.options__close}`}
+            onClick={() => setModalDisplay(false)}
```

Comment:
We should redirect the user to the "list" page when the request is correct. You should check the title of the modal to do this.
```suggestion
            onClick={() => {
              setModalDisplay(false);
              if (title !== 'Error') {
                 window.location.href = '/admins';
              }
            }}
```